files/es/tools/page_inspector/index.html

```diff
@@ -335,11 +335,9 @@ <h4 id="Cambio_de_diseño">Cambio de diseño</h4>
 
 <h2 id="Atajos_de_teclado">Atajos de teclado</h2>
 
-<p>{{ Page ("en-US/docs/tools/Keyboard_shortcuts", "page-inspector") }}</p>
```

Comment:
For instance, I'd suggest adding `<a href="/es/docs/Tools/Keyboard_shortcuts#inspector_de_páginas">Atajos de teclado</a>` (I don't speak Spanish, so I'm not sure if you'd prefer to change the link text :)) here to replace the macro

Reply:
[d'oh!](https://g.co/kgs/uXiMc5) I thought since it was deprecated it should be removed, I'll close this PR, it's not useful then

Your content link it's okay, like a Spanish native.

Reply:
Sorry if I was unclear — yes, the macro should be removed as it is indeed being deprecated. What I meant here was that in addition to deleting it, we should probably replace it with the link it's meant to lead to. Let me know if I should explain further :)

Reply:
No, that's ok, I understood, thank you